control-plane/store/src/lib.rs

```diff
@@ -1,2 +1,3 @@
 pub mod etcd;
 pub mod kv_store;
+pub mod v0;
```

Comment:
I wonder if this should be within another submodule, types::v0 or something

Reply:
Good suggestion - done.